plexpy/notification_handler.py

```diff
@@ -516,11 +548,26 @@ def build_notify_text(session=None, timeline=None, state=None):
                         'summary': metadata['summary'],
                         'tagline': metadata['tagline'],
                         'rating': metadata['rating'],
-                        'duration': duration
+                        'duration': duration,
+                        'stream_duration': stream_duration,
+                        'remaining_duration': duration - view_offset,
+                        'progress': view_offset,
+                        'progress_percent': progress_percent,
+                        'action': state
```

Comment:
Add `{action}` to the notification params modal popup (probably under stream details)  

Also, might as well add `{streams}` for the number of current streams.